rfcs/0002-typography-scaling.md

```diff
@@ -0,0 +1,162 @@
+We need our users to access our products anywhere, anytime. To make sure they can do so, we need to be present on almost every device possible especially on portables (mobiles & tablets). Every device offers a different experience and we need to make sure that we are flexible in that case. One of the major aspects of that experience is **readability** and therefore we are focusing on **typography scaling** across multiple devices.
+
+### **Use Cases**
+There can be several use-cases when a user wants to access the dashboard on multiple devices. Few of those can be,
+- **On Mobiles** (user can check for settlement numbers while commuting/traveling in a metro/bus)
+- **On Tablets** (user can capture a payment while on-the-go to their home back from the office)
+- **On Desktops** (user can request for access to the international payment via the dashboard)
+
+Or in-general a user can visit the company's marketing/landing page and consume information about us and our offerings.
+
+### **Expected Outcome**
+A typography system that is more readable/consumable across different devices (mobiles/tablets/desktops).
+
+# Detailed Design
+We are following a [**Major Second**](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) modular scale (1.125) for our typography system but and that currently has two major limitations,
+- **Does not cater for landing/marketing pages.** Which are generally much larger compared to the size-ratio on the dashboard (Maybe a [Major Third](https://type-scale.com/?size=14&scale=1.125&text=A%20Visual%20Type%20Scale&font=Lato&fontweight=400&bodyfont=Poppins&bodyfontweight=400&lineheight=1.65&backgroundcolor=white&fontcolor=%23333&preview=false) scale will work better). This is currently being looked by the comm. design team based on the same principles.
+- **Same type scale across all devices.** After the type scale is defined, we also need to define the scale across multiple device sizes. There are few approaches that we've came up with and after research from other DS/products like *GitHub*, *Eightshapes*, *Paystack*, *Stripe*, and a few more we've observed that for dashboards generally tweaking the **heading** sizes works best and the base font-size will remain same for all other devices/breakpoints.
+
+We'll look into the responsive type scale and see what approach fits best for our use case.
+
+### **Our Approach**
+>**Similar vertical & horizontal ratio across multiple breakpoints (for headings)**
+
+Our product is majorly used on mobiles/tablets/desktops. Therefore, We'll  consider only two breakpoints for the typography scale, one for mobiles & one for both desktops & tablets. Which is for devices having width `>=991px` (approx.)
+
+![Scaling: Similar vertical & horizontal ratio](https://miro.medium.com/max/2000/1*LpBKPTjjtW3I3xKEcae--Q.png)
```

Comment:
Shouldn't this image come before this section? Because right now if I read the below paragraph and then look at the image it feels disconnected.
>Our product is majorly used on mobiles/tablets/desktops. Therefore, We'll  consider only two breakpoints for the typography scale, one for mobiles & one for both desktops & tablets. Which is for devices having width `>=991px` (approx.)

Reply:
Also, why are you showing all breakpoints instead of only those that we are following?

Reply:
We will only have a few breakpoints, not as much as shown in the image. This is just to give an idea of how the scaling works. Breakpoint discussion is still pending from our end.